src/Lotto.js

```diff
@@ -1,18 +1,154 @@
+import { LOTTO_MESSAGES, LOTTO_NUMBER, LOTTO_PRIZE_MONEY } from "./constants/lotto.js";
+import { MissionUtils } from "@woowacourse/mission-utils";
+
 class Lotto {
-  #numbers;
+  #lottoNumbersStore = [];
+  #prizeMoney = 0;
+  #lottoResult = new Map([
+    [3, 0],
+    [4, 0],
+    [5, 0],
+    ["5B", 0],
+    [6, 0],
+  ]);
+
+  // 유효성 검사
+  #isLottoPriceValidate(price) {
+    if (price % LOTTO_NUMBER.DIVISION_PRICE === 0) {
+      return false;
+    }
+
+    return true;
+  }
+
+  #isLottoNumberLengthValidate(numbers) {
+    if (numbers.length === LOTTO_NUMBER.LENGTH) {
+      return false;
+    }
+
+    return true;
```

Comment:
요기도 `return`문 한줄로 축약 가능할 것 같아요!
```suggestion
    return numbers.length !== LOTTO_NUMBER.LENGTH;
```